assets/css/sensei-course-theme/mobile.scss

```diff
@@ -23,6 +17,7 @@
 
 		&--sidebar-open {
 			--header-height: 90px;
+			overflow: hidden;
```

Comment:
Just noting that we can move the Focus mode toggle block out of the sidebar if it makes life easier, it's only in there to look a bit more sensible in the editor.
It could even make sense as an up/down arrow in the header, left to the logo:

<img width="1208" alt="image" src="https://user-images.githubusercontent.com/176949/151462180-2a146492-864e-4d41-a8ae-181a0c65ee8e.png">

Reply:
I liked this idea! Implemented it here: https://github.com/Automattic/sensei/pull/4684/commits/ec632005c8f0779121be28414d7ebfac66fbf166

https://user-images.githubusercontent.com/876340/151560799-d80aaaca-6b82-4522-b86c-98d23664a605.mov